compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/PositionUtil.java

```diff
@@ -30,6 +30,12 @@
 class PositionUtil {
 
     static boolean withinBlock(LinePosition cursorPos, Location symbolPosition) {
+        /*
+        Temporary null check added
+         */
+        if (symbolPosition == null) {
```

Comment:
In what sort of scenarios was this null?

Reply:
This is being null when we are going to use the semantic API after the Desugar phase.